### ISSUE TITLE
Can I use linked cards?

### ISSUE THREAD

**Issue by enderman-2009** [cadergator10/opencomputer-security-system#37]
Is there a way to use linked card so i can use it across dimensions?

**Comment by cadergator10** [comment]
I assume you figured it out immediately. Yeah, linked cards work across dimensions. Just make sure that the server is chunkloaded as well as the computer in the other dimension

**Comment by enderman-2009** [comment]
could you make it so you can use link cards everywhere cause right now only door control computer supports it and i don't want to use wireless network cards because the sector control program isn't encrypted and the diagnostic tablet is the reason i can't use regular network cards

**Comment by cadergator10** [comment]
Yeah, the sector control program doesn't support, it, but the diagnostic tablet does. You MUST have both a network card and linking card though in any devices. If you have both a network card and linking card, all server messages go through the network card (traffic to one location) and all messages that need to be done locally (messages between door computers and the diagnostic tablet) are done through the network cards.

**Comment by enderman-2009** [comment]
yeah but the server is the one i don't want to put a wireless network card in cause it can be used to open any door that has a sector

**Comment by enderman-2009** [comment]
@cadergator10 since the sector control program isn't encrypted and that it can open doors if the server has a wireless network card and a door has a sector assigned then anyone that knows the port or just broadcasts on every port and knows the message can open that door

**Comment by enderman-2009** [comment]
also your explanation is very confusing you say it supports it but you say that both things go through netcard

**Comment by cadergator10** [comment]
> yeah but the server is the one i don't want to put a wireless network card in cause it can be used to open any door that has a sector

> @cadergator10 since the sector control program isn't encrypted and that it can open doors if the server has a wireless network card and a door has a sector assigned then anyone that knows the port or just broadcasts on every port and knows the message can open that door

While it's possible to only use linked cards for certain functions, for others its a little difficult to arrange for multiple devices. It's something I can add in the future for the sector control, but as for the moment I'm busy writing the server

> also your explanation is very confusing you say it supports it but you say that both things go through netcard

I'll try and rephrase it:
A device can either use the network card, or both a network card and a linking card. 
The linking card is only used for messages to be sent to the server and from the server to the device (eg: a door checking if a swiped card has correct credentials)
All other messages (like running Runtime Door Editing on a diagnostic tablet to the door) are communicated to with the modem.

Basically a device MUST have a wireless network card, and you also add the linking card. Not just a linking card alone, both

**Comment by enderman-2009** [comment]
the linked card doesn't work the server never receives the querry

**Comment by cadergator10** [comment]
Did you use the range extender program?

**Comment by enderman-2009** [comment]
i tryed with and without so unless im doing something wrong its broken

**Comment by enderman-2009** [comment]
nvm im just an idiot and forgot to connect it

**Comment by cadergator10** [comment]
I'll figure it out later. I dont have enough time today

**Comment by cadergator10** [comment]
Oh you fixed it. Ok good lol
